modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-vtpm-devices.adoc
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a vTPM device to a virtual machine (VM) stores secrets for that VM.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
```

Comment:
This first sentence strikes me as potentially hard to localize. I think you only need the second sentence to get this point across.